.github/workflows/rosa-scaling-benchmark-on-schedule.yml

```diff
@@ -0,0 +1,18 @@
+name: ROSA Cluster - Scaling Benchmark on Schedule
+
+on:
+  schedule:
+    - cron: '0 7 * * 1-5' # Runs At 07:00 UTC on every day-of-week from Monday through Friday.
+
+jobs:
+
+  checkout:
+    name: ROSA Scheduled Scaling Benchmark Run
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run AuthorizationCode scalability scenario
```

Comment:
This is not running only AuthorizationCode

Reply:
Oops, result of copy/paste. Resolved.